BackofficeBundle/DependencyInjection/Compiler/TwigGlobalsCompilerPass.php

```diff
@@ -28,6 +28,10 @@ public function process(ContainerBuilder $container)
             $formResources[] = 'OpenOrchestraBackofficeBundle:Form:form_div_layout.html.twig';
             $container->setParameter('twig.form.resources', $formResources);
         }
+
+        $container->setParameter('jms_serializer.handlers.datetime.default_format', \DateTime::ISO8601);
```

Comment:
yes so this line is useless no ? because the service jms_serializer.handlers.datetime is override